scenarios.js

```diff
@@ -1,438 +1,442 @@
+//special_rules should be treated with some kind of macro that recognises them and applies them when loading

```

Comment:
Yes, that and having multiple monster with the same deck per scenario (Guard Archer and Inox Archer) are TODO. Can add them as issues!